### DISCUSSION TITLE
Build 6 pages, 3 for Pidgin and 3 for Yoruba Bible

### DISCUSSION THREAD

**Discussion by sanmiajanaku** [DaBible-Foundation/DaBible-MainHub#171] [Ideas]
The following pages must be rebuilt for SEO purposes with nothing less than 300 to 600 words.
Each page must be assigned the role of Item Page on the Yoast SEO settings.

- [ ] Pidgin Audio Bible – Android App Version
- [ ] Pidgin Audio Bible – iPhone App Version
- [ ] Pidgin Audio Bible – Laptop App Version
- [ ] Yoruba Audio Bible – Android App Version
- [ ] Yoruba Audio Bible – iPhone App Version
- [ ] Yoruba Audio Bible – Laptop App Version

Thanks.

**Comment by emmanlex** [comment]
@sanmiayotunde what is the status of this task?
